README.md

```diff
@@ -1,5 +1,24 @@
-# Artemis 
+# Artemis
 
-This package provides extra functionality for containers 
+This package provides extra functionality for containers
 managed by Artemis.
 
+----------------
+
+# Example
+
+Install this package using:
+
+``` sh
+toit.pkg install artemis
```

Comment:
At the moment there is only one artemis package, but once there are more, this would be ambiguous.
```suggestion
toit.pkg install github.com/toitware/toit-artemis
```